### PR TITLE
Skip reboot after update package for sles15sp2+ host

### PR DESCRIPTION
**Pull request by Julie-CAO**
This PR is associated with https://github.com/SUSE/qa-automation/pull/749.

[gi-guest_developing-on-host_developing-xen](http://10.67.129.51/tests/1624)
[gi-guest_developing-on-host_sles15sp2-xen](http://10.67.129.51/tests/1625)
[gi-guest_developing-on-host_sles12sp5-kvm](http://10.67.129.51/tests/1626)

@alice-suse @guoxuguang @waynechen55 Please help review.

### CODE REVIEW

File: products/sle/main.pm [hunk 1]
```diff
@@ -769,7 +769,7 @@ elsif (get_var("VIRT_AUTOTEST")) {
         }
         loadtest "virt_autotest/install_package";
         loadtest "virt_autotest/update_package";
-        loadtest "virt_autotest/reboot_and_wait_up_normal";
+        loadtest "virt_autotest/reboot_and_wait_up_normal" unless is_sle('>=15-sp2');
```
Comment (alice-suse) [inline products/sle/main.pm:772]: I am afraid reboot is only not necessary for developing product.

Comment (Julie-CAO) [reply products/sle/main.pm]: Yes, I think so. change to apply to developing only and verify

only verify the scheduled modules:
[gi-guest_developing-on-host_developing-xen](http://10.67.129.51/tests/1627)
[gi-guest_developing-on-host_sles15sp2-xen](http://10.67.129.51/tests/1628)
[gi-guest_developing-on-host_sles12sp5-kvm](http://10.67.129.51/tests/1629)

**Comment by alice-suse** [comment]
@Julie-CAO can you fix the travis failure?

**Comment by Julie-CAO** [comment]
> @Julie-CAO can you fix the travis failure? Code LGTM.

The travis check just passed. Thanks!

**Pull request merged**